core/templates/pages/story-editor-page/editor-tab/story-editor.component.html

```
@@ -199,33 +199,106 @@ <h3 class="story-card-header">Story Card</h3>
+                          <span class="arc-boundary-title">
+                            {{ getArcForNode(node.getId())?.getTitle() }}
+                          </span>
+                          <span class="arc-boundary-description" *ngIf="getArcForNode(node.getId())?.getDescription()">
+                            {{ getArcForNode(node.getId())?.getDescription() }}
+                          </span>
+                          <span class="arc-boundary-chapter-count">
+                            {{ getArcForNode(node.getId())?.getNodeIds()?.length }} chapters
+                          </span>
+                        </div>
+                        <div class="arc-boundary-actions">
+                          <button class="btn btn-sm arc-edit-button"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)